tasks/__init__.py

```diff
@@ -776,8 +776,6 @@ def packages(ctx):
         'install elasticsearch',
         'install rabbitmq',
         'install node',
-        'tap tokutek/tokumx',
```

Comment:
See my original commit and briang's comment. That's what I originally had more or less.